board/ember/linux_defconfig

```diff
@@ -0,0 +1,476 @@
+CONFIG_SYSVIPC=y
```

Comment:
This file looks huge ... is it really a saved `make linux-update-defconfig` file or a saved `.config`?  To be able to track changes between different archs, and try to keep them feature wise in sync, we used to save in defconfig style instead, which is a LOT smaller.

I may be wrong here, so treat this as an fyi and minor reflection from my side.

Reply:
It is pretty much a copy of the envoys linux_defconfig but perhaps that should be looked in to as well.

Reply:
Yeah, please do, for your own sake.  Iirc that too is wrong.  Going forward you'll be much happier with the minimized version as the origin :)